snips/run.sh

```diff
@@ -5,13 +5,40 @@ CONFIG_PATH=/data/options.json
 
 MQTT_BRIDGE=$(jq --raw-output '.mqtt_bridge.active' $CONFIG_PATH)
 ASSISTANT=$(jq --raw-output '.assistant' $CONFIG_PATH)
+LANG=$(jq --raw-output '.language' $CONFIG_PATH| awk -F '-' '{print $1}')
+CUSTOMTTS=$(jq --raw-output '.custom_tts' $CONFIG_PATH)
+PLATFORM=$(jq --raw-output '.tts_platform' $CONFIG_PATH)
+
+if [ "$HOSTTYPE" == "x86_64" ]; then
+    ARCH="amd64"
+else
+    ARCH="armhf"
+fi
+
+echo "[INFO] ARCH: $ARCH"
+echo "[INFO] LANG: $LANG"
 
 echo "[INFO] Checking for /share/snips.toml"
 if [ -f "/share/snips.toml" ]; then
-    echo "[INFO] Installing /share/snips.toml"
+    echo "[INFO] - Installing /share/snips.toml"
     cp -v /share/snips.toml /etc/
 fi
 
+if [ "$CUSTOMTTS" == "true" ]; then
+    if [ -z "$PLATFORM" ]; then
+        echo "[ERROR] - platform must be set to use custom tts!"
+    else
+        echo "[INFO] - Using custom tts"
+        echo "provider = \"customtts\"" >> /etc/snips.toml
+        echo "customtts = { command = [\"/usr/bin/customtts.sh\", \"$PLATFORM\", \"%%OUTPUT_FILE%%\", \"$LANG\", \"%%TEXT%%\"] }" >> /etc/snips.toml
+    fi
+else
+    echo "[INFO] - Using default tts (picotts)"
+fi
+
+echo "[INFO] Copying mpg123 for $ARCH"
+mv -v /usr/bin/mpg123-$ARCH /usr/bin/mpg123
```

Comment:
Do that inside Dockerfile

Reply:
Sure. The issue with a package is that the snips platform is actually based on debian so we would need to add a fair number of dependencies. I actually compiled this from source to get a static binary just to make it easier. I could probably set it up in the docker file to install it via apt as part of the build process.